### DISCUSSION TITLE
Add stamina cost info to melee weapon descriptions

### DISCUSSION THREAD

**Discussion by metalheimer** [cataclysmbn/Cataclysm-BN#2797] [General]
### Is your feature request related to a problem? Please describe.

This probably should've existed since day one. It would certainly help compare melee weapons.

### Describe the solution you'd like

Two additional lines to descriptions, suggested below the "moves per attack" line.

Stamina cost (base): 123
Stamina cost (with modifiers): 132

or Stamina cost (actual): 132

### Additional context

Include stamina info in character sheet. Max stamina, base stamina regen rate and current modified stamina regen rate.

**Comment by Ignaramico** [comment]
idk, i feel that is like a good idea, but not that necessary, i think that if there is going to be more info cluttered in the item description/comparation it should be the hit chance

**Comment by scarf005** [comment]
https://github.com/cataclysmbnteam/Cataclysm-BN/blob/a5e79b74f5431ce655056373c606588e0ab19371/src/melee.cpp#L653-L664

might be tricky to implement. _proceeds to blame spaghetti code_
